lib/config/alias.inc

```diff
@@ -18,6 +18,6 @@ ALIAS   --math-bbc              -Cc-fp-mode=z88     -lbbc_math
 #
 # Math library aliases (classic + newlib)
 #
-ALIAS   --math16                -Cc-fp-mode=ieee16  -lmath16 -Cc-D__MATH_MATH16 -D__MATH_MATH16 -pragma-define:CLIB_16BIT_FLOAT=1
```

Comment:
@feilipu I think this can be removed - all that's needed now is -lmath16?

Reply:
I'd like to leave the alias for consistency, but I agree all the definitions can depart.

Reply:
Actually, it is nice to be able to choose whether code should use the normal path or the 16 bit path, based on whether the library is included. Will keep the definitions, unless there's an issue?